autoload/quickrun.vim

```diff
@@ -497,6 +497,11 @@ let g:quickrun#default_config = {
 \   'command': 'xcrun',
 \   'exec': ['%c swift %s'],
 \ },
+\ 'tmux': {
+\   'command': 'tmux',
+\   'cmdopt': 'source-file',
```

Comment:
`cmdopt` exists for user's customization.
Can a user become happily by changing this?
If no, you can move this to `exec`.